src/dynamics/video_player/player/player.js

```diff
@@ -580,10 +594,94 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                         if (Types.is_boolean(c)) return "10px";
                         return (Types.is_string(c) ? parseFloat(c.replace(/\D/g, '')).toFixed() : c) + 'px';
                     },
-                    "sidebar_active:is_floating,with_sidebar,showgallery,fullscreened": function(isFloating, withSidebar, showGallery, fullscreened) {
+                    "sidebar_active:with_sidebar,showsidebargallery,playlist": function(withSidebar, showSidebarGallery, playlist) {
+                        if (!showSidebarGallery && Types.is_defined(this.get("initialoptions.nextwidget"))) {
+                            this.set("nextwidget", this.get("initialoptions.nextwidget"));
+                        }
+                        return withSidebar || (showSidebarGallery && playlist && playlist.length > 0);
+                    },
+                    "show_sidebar:sidebar_active,is_floating,with_sidebar,fullscreened,mobileviewport": function(sidebarActive, isFloating, withSidebar, fullscreened, mobileViewport) {
                         if (fullscreened) return false;
-                        return (isFloating && withSidebar) || (showGallery && !isFloating);
-
+                        this.set("floatingsidebar", sidebarActive && isFloating && withSidebar);
+                        this.set("gallerysidebar", sidebarActive && !isFloating && (Types.is_defined(mobileViewport) && !mobileViewport));
+                        if (this.get("gallerysidebar")) {
+                            if (!this.get("nextwidget") && Types.is_undefined(this.set("initialoptions.nextwidget"))) {
+                                this.set("initialoptions.nextwidget", false);
+                            }
+                            this.set("nextwidget", true);
+                        }
```

Comment:
Hi @sambua, why are we forcing the next widget to be turned on with the gallery sidebar? We should be able to configure those separately.
I also think the value stored on `initialoptions.nextwidget` might override settings that were applied after player creation.

Reply:
Hi @ntshcalleia as it's acting like nextwidget (with all events etc.) and when sidebar is active we're not showing nextwidget, but using all it's features. And when sidebar will not be active it will set initial `nextwidget` what was set by user initially.

Reply:
Hi @sambua, what I mean is that we want to be able to show the gallery sidebar without next widget features. Just have it listing the next videos so that the user can click on one, without having it starting countdown after a fixed amount of time.

Reply:
But I can't disable them with nextwidget: false. I should be able to do that.

Reply:
Why you can't? it will be turned on when sidebar is active and will turn false as soon as it will be not visible.

Reply:
Hi @sambua, if I set it to false I don't want it to be turned on when sidebar is active. I'm talking about functionality, when sidebar gallery is visible `nextwidget: false` should disable countdown and stay button.

Reply:
Ah I see now what you meant. I've added a new sidebar options option as `autonext`, setting false will turn off nextwiget on sidebar gallery side as well, and not allow switch to the next video.